src/activities/bargame/ActivityInfo.qml

```diff
@@ -0,0 +1,35 @@
+/* GCompris - ActivityInfo.qml
+ *
+ * Copyright (C) 2016 UTKARSH TIWARI <iamutkarshtiwari@kde.org>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+import GCompris 1.0
+
+ActivityInfo {
+  name: "bargame/Bargame.qml"
+  difficulty: 1
+  icon: "bargame/bargame.svg"
+  author: "Utkarsh Tiwari &lt;iamutkarshtiwari@kde.org&gt;"
+  demo: true
+  title: qsTr("Bargame (against Tux)")
+  description: ""
+  //intro: "put here in comment the text for the intro voice"
```

Comment:
can you put here the good text (check in http://gcompris.net/wiki/Voices_translation to have the one needed)?